validator/schemas/summary-metrics-schema-v1.json

```diff
@@ -128,7 +133,7 @@
           "key"
         ],
         "properties": {
-          "select": {
+          "key": {
```

Comment:
Does this require modifying existing files?

Reply:
No, luckily all existing files are valid, so this change has no effect, but before we were basically not validating tags metrics.